### PR TITLE
Create a chrome extension which disables multiple routes for webrtc

### PR DESCRIPTION
**Pull request by guoweis**
a barebone extension which disables webrtc's multiple_routes. Once installed, the local host candidate will not be collected as part of address discovery phase of ICE.

More change is expected to meet the requirement of an official chrome extension.

### CODE REVIEW

**Comment by guoweis** [comment]
a barebone extension which disables webrtc's multiple_routes. Once installed, the local host candidate will not be collected as part of address discovery phase of ICE.

**Comment by guoweis** [comment]
I signed it!

File: samples/extensions/multipleroutes/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This extension, once installed, will disable multiple routes option in WebRTC. This will prevent any website from accessing local machine's private IP addresses or public ones which should have been hidden by the usage of VPN. The only address left for WebRTC will be the same one used for http traffic.
```
Comment (pthatcherg) [inline samples/extensions/multipleroutes/README.md]: "will disable multiple routes option in WebRTC"  might be better as "will disable the option for multiple network routes in WebRTC

Comment (pthatcherg) [inline samples/extensions/multipleroutes/README.md]: "accessing local machine's private IP addresses" => "accessing the local machine's private IP addresses"

File: samples/extensions/multipleroutes/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Note that the installation of this extension could have negative impact on WebRTC real time media as the route might not be the optimized one. Worst case, if no Stun/Turn server is specified, WebRTC will fail completely as there is no local address for remote peer to connect to.
```
Comment (pthatcherg) [inline samples/extensions/multipleroutes/README.md]: Stun/Turn => STUN/TURN
remote peer => the remote peer

File: samples/extensions/multipleroutes/manifest.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "manifest_version": 2,
+  "name": "WebRTC extension to disable multiple routes",
```
Comment (pthatcherg) [inline samples/extensions/multipleroutes/manifest.json]: "Extension to disable the option for multiple routes in WebRTC" might be a little better.

**Comment by guoweis** [comment]
I signed it!

**Comment by pthatcherg** [comment]
lgtm

**Comment by KaptenJansson** [comment]
@guoweis anything left to do? Like change the name of it?

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**